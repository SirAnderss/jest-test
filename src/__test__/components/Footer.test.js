import React from 'react';
import { mount } from 'enzyme';
import Footer from '../../components/Footer';
import { create } from 'react-test-renderer';

describe('<Footer />', () => {
  const footer = mount(<Footer />);
  test('Render Footer component', () => {
    expect(footer.length).toEqual(1);
  });

  test('Title render', () => {
    expect(footer.find('.Footer-title').text()).toEqual('Platzi Store');
  });
});

describe('Footer Snapshot', () => {
  test('Test UI Footer component', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON).toMatchSnapshot();
  });
});
