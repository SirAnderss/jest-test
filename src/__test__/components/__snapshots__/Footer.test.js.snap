// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot Test UI Footer component 1`] = `[Function]`;
